rocAL/rocAL/include/meta_data/meta_node_resize_crop_mirror.h

```diff
@@ -37,8 +37,8 @@ class ResizeCropMirrorMetaNode:public MetaNode
     private:
         void initialize();
         std::shared_ptr<RocalCropParam> _meta_crop_param;
-        unsigned int _dst_width, _dst_height, _crop_w, _crop_h;
-        vx_array _x1, _y1, _x2, _y2, _mirror;
+        unsigned int  _crop_w, _crop_h;
+        vx_array _x1, _y1, _x2, _y2, _dst_width, _dst_height, _mirror;
         std::vector<uint> _x1_val, _y1_val, _x2_val, _y2_val, _mirror_val;
         float _dst_to_src_width_ratio, _dst_to_src_height_ratio;
```

Comment:
All these variables should they be private?
Why not change scope to local?

Is _dst_width and _dst_height used anywhere?